packages/ranges/src/elements/MultiThumbRange.js

```diff
@@ -16,11 +16,15 @@ import { KEY_CODES } from '@zendeskgarden/react-selection';
 import { withTheme, isRtl, getDocument } from '@zendeskgarden/react-theming';
 import RangeStyles from '@zendeskgarden/css-forms/dist/range.css';
 
+const COMPONENT_ID = 'ranges.multi_thumb_range';
+
 /**
  * These Styled components are not exported with the other Views due to their logic
  * being more tightly coupled with this specific implemenation.
  */
 const StyledSlider = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
```

Comment:
This should be added to all views right? Otherwise I won't be able to theme it like I can with the single range.

Reply:
Originally thought we would wan't these theme-able, but you're right, it should be possible.

I've added unique keys to each styled component as well as the `retrieveTheme` injectable.